target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -18,6 +18,7 @@ board_config_update
 
 case $board in
 11acnas|\
+afoundry-ew1200|\
```

Comment:
I based my porting on the ZBT WG2626 as it is the closest device to the one I have in terms of specs. And for that device, board name is prefixed with zbt- everywhere, which is why I prefixed my board with afoundry-
Regardless, I got the point and will amend my commit to remove the prefixes everywhere.